lib/screens/globalChallenge.dart

```diff
@@ -165,7 +178,8 @@ class _GlobalChallengeState extends State<globalChallenge> {
 
       final capitalizedWords = words.map((word) {
         if (word.isNotEmpty) {
-          return word[0].toUpperCase() + word.substring(1);
+          return word[0].toUpperCase() +
+              word.substring(1).trim(); // trim shld remove whitespaces
```

Comment:
Not sure why, but this line didn't remove all the whitespace from an answer (see image)
<img width="203" alt="Screenshot 2023-11-12 at 9 19 13 PM" src="https://github.com/dartmouth-cs98-23f/verbatim_frontend/assets/76986782/386766e9-0378-44d0-8a3c-9021cee4068f">